docs/sdk-and-tools/elastic-search.md

```diff
@@ -0,0 +1,35 @@
+---
+id: elastic-search
+title: Elastic Search
```

Comment:
Elasticsearch instead of Elastic Search in all over the places

Reply:
done